src/express/commands/send-stake-update.js

```diff
@@ -102,22 +105,34 @@ export async function sendStakeUpdateEvent(validatorID) {
   console.log('Old Validator Power:  ' + oldValidatorPower)
 
   // Adding 100 MATIC stake
-  tx = stakeManagerContract.methods.restake(
-    validatorID,
-    rootChainWeb3.utils.toWei('100'),
-    false
-  )
-  signedTx = await getSignedTx(
-    rootChainWeb3,
-    StakeManagerProxyAddress,
-    tx,
-    validatorAccount,
-    pkey
-  )
-  const Receipt = await rootChainWeb3.eth.sendSignedTransaction(
-    signedTx.rawTransaction
-  )
-  console.log('Restake Receipt txHash:  ' + Receipt.transactionHash)
+  let command = `export PATH="$HOME/.foundry/bin:$PATH" && cast send ${MaticTokenAddr} "transfer(address,uint256)" ${validatorAccount} 100000000000000000000 --rpc-url http://localhost:9545 --private-key ${fundingKey}`
```

Comment:
I would add some constants in anvil utils to define all the methods we need.
e.g
```
export const TRANSFER_METHOD = transfer(address,uint256)
```

Reply:
Also, shall we parametrize localhost to use `machine[0]`?